spotbugsTestCases/src/fakeAnnotations/org/apache/avro/reflect/Nullable.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
SpotBugs doesn't belong to Apache. Why is this copyright notice in use?

Reply:
Sorry for the slow response

I copied the file from the Apache avro source, following what seems to have been done for the `android.support.annotation` classes (which use the same license).

I'm no expert on how this license works; if you think it's more reasonable to remove the license block here, I'd be happy to do so.

Reply:
Ah. That does make sense.